src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version},`
+      + ` wheels: ${wheels}`;
```

Comment:
```suggestion
      return `name: ${this.name},`
      + ` chip version: ${this.version},`
      + ` wheels: ${wheels}`;
```
Так виглядатиме трохи гарніше